include/fw_info.h

```diff
@@ -64,10 +72,10 @@ struct __packed fw_info {
 	u32_t firmware_address;
 
 	/* Where to place the getter for the ABI provided to this firmware. */
-	fw_info_abi_getter *abi_in;
+	struct fw_info_abi_getter_info *abi_in;
 
 	/* This firmware's ABI getter. */
-	const fw_info_abi_getter abi_out;
+	fw_info_abi_getter abi_out;
```

Comment:
Why is this no longer `const` ?

Reply:
Because none of the other struct members are const. All relevant struct instantiations are marked const anyways, so this isn't strictly needed.